baseband/vlbi_base/base.py

```diff
@@ -188,19 +204,20 @@ def tell(self, unit=None):
         if unit == 'time':
             return self.start_time + self.tell(unit=u.s)
 
-        return (self.offset * u_sample).to(unit, equivalencies=[(u.s, u.Unit(
-            self.samples_per_frame * self.frames_per_second * u_sample))])
+        return (self.offset / self.sample_rate).to(unit)
 
     def _frame_info(self):
         offset = (self.offset +
                   self.header0['frame_nr'] * self.samples_per_frame)
+        framerate = int(np.round((
```

Comment:
Really trivial, but maybe move last parenthesis to next line, to make it clearer what is being rounded.